app/screens/channel/channel.tsx

```diff
@@ -121,10 +121,12 @@ const Channel = ({channelId, componentId, displayName, isOwnDirectMessage, membe
                     title={title}
                 />
                 <View style={[styles.flex, {marginTop}]}>
+                    {Boolean(channelId) &&
```

Comment:
When may this happen? Or is it just a sanity check?

Reply:
here https://github.com/mattermost/mattermost-mobile/blob/gekidou/app/actions/local/channel.ts#L45